test/Test-EndToEnd.js

```diff
@@ -13,9 +13,9 @@ const { Storage } = require('@google-cloud/storage');
 const TUS_RESUMABLE = require('../lib/constants').TUS_RESUMABLE;
 
 const STORE_PATH = '/test/output';
-const PROJECT_ID = 'tus-node-server';
-const KEYFILE = path.resolve(__dirname, '../keyfile.json');
-const BUCKET = 'tus-node-server-ci';
+const PROJECT_ID = 'supervisor-3dsurvey';
+const KEYFILE = '/home/mitjap/Projects/rune-panorama/supervisor-3dsurvey-editor-b9c28cd4108c.json';
+const BUCKET = 'mitjap-test-tus-node-server';
```

Comment:
We should change this back. Ideally these values come from env as well so we can use `npm run test:local` locally and CI will inject the vars so nothing has to change in the codebase